lib/cocoapods/command/spec/lint.rb

```diff
@@ -97,7 +97,7 @@ def podspecs_to_lint
         end
 
         def podspecs_tmp_dir
-          Pathname(File.join(Pathname.new('/tmp').realpath, '/CocoaPods/Lint_podspec'))
+          Pathname.new(Dir.tmpdir) + '/CocoaPods/Lint_podspec'
```

Comment:
When using + with Pathname's, there shouldn't be a slash at the start of the second path.

```
irb(main):012:0> Pathname.new(Dir.tmpdir) + '/CocoaPods/Lint_podspec'
=> #<Pathname:/CocoaPods/Lint_podspec>
irb(main):013:0> Pathname.new(Dir.tmpdir) + 'CocoaPods/Lint_podspec'
=> #<Pathname:/var/folders/tj/rgvrw2vj6ygbxvg2kd8f0gl00000gn/T/CocoaPods/Lint_podspec>
```